src/App.js

```diff
@@ -1,5 +1,19 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES } from "../constants/messages.js";
+import playGame from "./playGame.js";
+import askForRestart from "../utils/askForRestart.js";
 class App {
-  async play() {}
+  async play() {
+    let continueGame = true;
```

Comment:
```suggestion
    let isContinueGame = true;
```
`boolean` 값이기 때문에 네이밍으로 `is`, `has`, `should`와 같은 접두사를 붙여도 좋을 것 같아요~!

Reply:
사소한 부분인데 정말 자세히 봐주셨네요 감사합니다!!😃